scripts/get_deployed_brackets.js

```diff
@@ -0,0 +1,34 @@
+const FleetFactory = artifacts.require("FleetFactory")
+
+const argv = require("./utils/default_yargs").option("masterSafe", {
+  type: "string",
+  describe: "Address of Gnosis Safe owning every bracket",
+  demandOption: true,
+}).argv
+
+module.exports = async (callback) => {
+  try {
+    const fleetFactory = await FleetFactory.deployed()
+    const events = await fleetFactory.getPastEvents("FleetDeployed", {
+      filter: { owner: argv.masterSafe },
+      fromBlock: 0,
+      toBlock: "latest",
+    })
+    const bracketsAsObjects = events.map((object) => object.returnValues.fleet)
+    const bracketAddresses = [].concat(...bracketsAsObjects)
+
+    // writing the brackets into a csv file
+    const createCsvWriter = require("csv-writer").createObjectCsvWriter
+    const csvWriter = createCsvWriter({
+      path: "./bracket-addresses.csv",
+      header: [{ id: "bracketAddress", title: "bracket-address" }],
+    })
+    const bracketsFormatted = bracketAddresses.map((bracketAddress) => [{ bracketAddress: bracketAddress }])
+    const records = [].concat(...bracketsFormatted)
```

Comment:
Why is this line needed? As I understand it it's a way to create a shallow copy of `bracketsFormatted`, but even if `csvWriter.writeRecords` were to change the object you don't need it anymore.